src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -81,7 +81,7 @@ const DragDropcontext: FC<dragDropProps> = ({
           ? result.draggableId
           : result.combine.draggableId;
         const data = {
-          status: ASSIGNED,
+          status: ACTIVE,
```

Comment:
Status must be set to 'active' else the assigned member will not be removed from the idle members list

Reply:
Then we'll have to make a new issue in the backend to change the logic for IdleMembers.
Because a user can have multiple tasks ASSIGNED to them but they'll only work actively on one task.
Which will have the status ACTIVE.

Refer [this](https://discord.com/channels/673083527624916993/735063616168001536/848892753004986369) discussion

Reply:
Right, But for the availability panel, we need to do something to handle both